minemeld/ft/taxii.py

```diff
@@ -57,6 +60,9 @@
 from . import actorbase
 from .utils import dt_to_millisec, interval_in_sec, utc_millisec
 
+
+del stix_edh  # stix_edh is imported to register the EDH data marking extensions, but it is not directly used
```

Comment:
Any particular reason we want to delete it even if not being used directly?

Reply:
This silences a warning about the import being unnecessary and prevents the 'Optimize Imports' in PyCharm from removing the import.

Reply:
please could you add this note to the comment on the line? for posterity